modules/contrib/cyber_bridge/LICENSE

```diff
@@ -0,0 +1,59 @@
+SIMULATOR SOFTWARE LICENSE AGREEMENT
```

Comment:
Not sure if we are compatible with this. I'll leave it to @jinghaomiao @jmtao

Reply:
I have no idea.  let's see if @jinghaomiao may has any input. otherwise, we'll have to consult legal.

Reply:
I have replaced the license with Apache 2.0.